cmake/configs/aarch64/linux/config.cmake

```diff
@@ -12,8 +12,8 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_GNU_SOURCE" CACHE STRING "" FORCE)
 # Linux requires buffers aligned to 4KB boundaries for O_DIRECT to work.
 set(WT_BUFFER_ALIGNMENT_DEFAULT "4096" CACHE STRING "")
 
-# ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC instructions.
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crc" CACHE STRING "" FORCE)
+# ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC and RCpc instructions.
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8.3-a+crc" CACHE STRING "" FORCE)
```

Comment:
Is it commercially acceptable to disavow previous armv8 ISA versions?

Reply:
I'll need to discuss with the wider team, I don't know if we can blanket prevent it. Which is partly why I liked the configuration bool.